Sources/TuistCore/GraphTraverser/GraphTraversing.swift

```diff
@@ -74,6 +76,8 @@ public protocol GraphTraversing {
     /// - Returns: All direct and transitive target dependencies
     func allTargetDependencies(path: AbsolutePath, name: String) -> Set<GraphTarget>
 
+    func allTargetDependencies(_ rootTargets: [GraphTarget]) -> Set<GraphTarget>
```

Comment:
Is this traversing from those targets? If so:
```suggestion
    func allTargetDependencies(traversingFromTargets: [GraphTarget]) -> Set<GraphTarget>
```